src/ngx_http_lua_util.c

```diff
@@ -1892,13 +1893,13 @@ ngx_http_lua_escape_uri(u_char *dst, u_char *src, size_t size, ngx_uint_t type)
         0xffffffff, /* 1111 1111 1111 1111  1111 1111 1111 1111 */
 
                     /* ?>=< ;:98 7654 3210  /.-, +*)( '&%$ #"!  */
-        0xfc00886d, /* 1111 1100 0000 0000  1000 1000 0110 1101 */
+        0x80000029, /* 1000 0000 0000 0000  0000 0000 0010 1001 */
```

Comment:
Do we need to update the comment on line 1890 above accordingly? Also, why this change? Is it for sync'ing upstream nginx core changes?

Reply:
yes